inc/class-job.php

```diff
@@ -81,8 +84,11 @@ public function mark_completed() {
 	}
 
 	public function reschedule() {
-		$this->nextrun = date( MYSQL_DATE_FORMAT, strtotime( $this->nextrun ) + $this->interval );
-		$this->status  = 'waiting';
+		$date = new DateTime( $this->nextrun, new DateTimeZone( 'UTC' ) );
+		$date->add( new DateInterval( 'PT' . $this->interval . 'S' ) );
```

Comment:
Super minor micro-optimisation for the string concatenation here
```suggestion
		$date->add( new DateInterval( "PT{$this->interval}S" ) );
```